### PR TITLE
Combine runctestimage and runcimage

### PR DESCRIPTION
**Pull request by hqhq**
There is no need that keep them separate, it also
fixes #1006.

Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

**Comment by hqhq** [comment]
We'll have to change Jenkins config to adapt this change, @crosbymichael @mrunalp anyone can do this?

**Comment by cyphar** [comment]
lgtm once Janky is reconfigured.

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,46 @@
-FROM runc_test
+FROM golang:1.6.2
```
Comment (mrunalp) [inline Dockerfile]: Can we use 1.7?

**Comment by hqhq** [comment]
@mrunalp Updated.

**Comment by mlaventure** [comment]
Temporary updated Jenkins configuration to run Janky on this.

**Comment by crosbymichael** [comment]
LGTM

**Comment by cyphar** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Comment by mlaventure** [comment]
Updated Jenkins permanently :tada:

**Comment by cyphar** [comment]
Uhm, this is causing issues for me now. It's because they made the `ENTRYPOINT` of `golang:1.7.0` to be non-empty:

```
docker: Error response from daemon: oci runtime error: exec: "/tmpmount": stat /tmpmount: no such file or directory.
```

**Pull request merged**